src/Symfony/Component/Form/Extension/Core/EventListener/FixUrlProtocolListener.php

```diff
@@ -36,7 +36,7 @@ public function onSubmit(FormEvent $event)
     {
         $data = $event->getData();
 
-        if ($this->defaultProtocol && $data && \is_string($data) && !preg_match('~^[\w+.-]+://~', $data)) {
+        if ($this->defaultProtocol && $data && \is_string($data) && !preg_match('~^[\w+.-]+://~', $data) && false === strpos($data, '/')) {
```

Comment:
A recent PR for v4.4 replaces using « str_contains() »
Maybe leveragr the same way

Reply:
Done